templates/webpay/plus_mall_deferred/capture.html

```diff
@@ -0,0 +1,28 @@
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <title>Ejemplo Webpay Plus Mall Deferred- Capturar Transacci&oacute;n</title>
```

Comment:
Probably better to keep this as "Diferido" instead of "Deferred", so people don't have to second guess themselves